ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1449,6 +1450,19 @@ ip_routing: < true | false >
 ipv6_unicast_routing: < true | false >
 ```
 
+#### Router General configuration
+
+```yaml
+router_general:
+  vrfs:
+    < destination-vrf >:
+      leaks:
+        - source_vrf: < source-vrf >
+          policy: < route-map policy >
+        - source_vrf: < source-vrf >
+          policy: < route-map policy >
```

Comment:
I would recommend updating `policy:` to `subscribe_policy:` to match cli.

Reply:
Fixed in 7b15495